image-customize

```diff
@@ -143,30 +143,33 @@ parser.add_argument('-n', '--no-network', action='store_true', help='Do not conn
 parser.add_argument('image', help='The image to use (destination name when using --base-image)')
 args = parser.parse_args()
 
+if not (args.run_command or args.install or args.script or args.upload or args.resize):
```

Comment:
For what it's worth, I've never liked how this works.  I'd sort of prefer if we built a single array and executed the operations in the order they were given, like how `vm-run` does with `-s` and `-e`.  In the case of `vm-run` the implementation isn't beautiful, but it's conceptually better.

Reply:
Right, I can't say I like it much either. There are good reasons to run a command before and after installing packages, and it's totally not obvious why --run-command runs before, but --script runs after. (It usually makes sense in practice, but being explicit is better). argparse does not make this easy, but I'll add it to my todo list to experiment with that.